src/classes/CRLP_Operation_TEST.cls

```diff
@@ -55,4 +55,112 @@ private class CRLP_Operation_TEST {
         System.assertEquals(CRLP_Operation.RollupType.SUM, CRLP_Operation.getRollupTypeFromString('Sum'));
     }
 
+    /**
```

Comment:
All of the following test methods test there is a 1 to 1 correspondence between the SObjectField's Picklist Values and a Enum.
-  `Rollup__mdt.Operation__c` and `CRLP_Operation.RollupType`    
    -  `assertRollupTypeExistsForEachRollupOperationPicklistValue`
    -  `assertRollupOperationPicklistValueExistsForEachRollupType `
-  `Rollup__mdt.Time_Bound_Operation_Type__c` and `CRLP_Operation.TimeBoundOperationType`
    -  `assertTimeBoundOperationTypeExistsForEachRollupTimeBoundOperationTypePicklistValue `
    -  `assertRollupTimeBoundOperationTypePicklistValueExistsForEachTimeBoundOperationType `

Reply:
No longer used